.circleci/scripts/git-diff-develop.ts

```diff
@@ -82,9 +95,11 @@ async function fetchUntilMergeBaseFound() {
  */
 async function gitDiff(): Promise<string> {
   await fetchUntilMergeBaseFound();
-  const { stdout: diffResult } = await exec(`git diff --name-only origin/HEAD...${process.env.CIRCLE_BRANCH}`);
```

Comment:
I'm noticing a problem here: we're getting the base branch, not the base commit. This is problematic because the results are now dependent upon the exact timing of when this runs, and they're comparing changes made to develop that aren't in the target branch yet.

We're using this diff as a measure of what has changed in the PR, but because we're not comparing with the correct commit, we're going to get skewed results (as changes are made to `develop`, or as `develop` is merged into the PR, etc.). This problem is in the original implementation as well though, not introduced here.

I will follow up on this separately. We have fixed this same problem already elsewhere, we can fix it here too without much effort.

Reply:
Given that this is just used to run modified tests additional times, I'm having a hard time imagining a scenario where this has a negative impact, despite it being technically incorrect. I'm thinking maybe this is fine, not worth fixing for now.